notifikasjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/db/AapenSak.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.notifikasjon.db
+
+import org.jetbrains.exposed.sql.Table
+import org.jetbrains.exposed.sql.javatime.datetime
+
+object AapenSak : Table("aapen_sak") {
```

Comment:
Åpen Sak og Apen ID, samt ÅpenRepo -- dette navnet / prefixet skurrer for meg.
 Åpen gir meg assosiasjoner til noe som står åpent, f eks en sak som ikke er avsluttet… Da blir det ekstra rart når feks en åpen sak settes til avsluttet med en gang. Synes AGI er et bedre navn / prefix, selv om det er et akronym, men vi kan gjerne se om vi kan finne noen andre alternativer.

Reply:
Uønsket / Uventet / Overraskelse! Er alle sammen litt dekkende, men blir også litt feil :D 
La meg tenke litt til...

Reply:
vilkårlig / ivrig / uberegnelig / ubedt / ubekreftet / tilfeldig.. Nei, hva med: 
Uoppfordret, selvpålagt eller selvbestemt - de siste her er vel bedre..

Reply:
Selvbestemt liker jeg! De andre (og alt jeg har klart å komme på selv) er upresise og perer breg av at vi ikke vil ha inntektsmeldinger, men selvbestemt innkapsulerer det hele på en god måte.

Reply:
JA til selvbestemt IM! :D

Reply:
De vitsene får vi bare leve med :D

Reply:
Jeg tenker at å bytte fra "åpen" til "selvbestemt" er utenfor scope til denne PR-en, så om det er eneste mangel så merger jeg den gjerne.

Vi kan ta en runde med resten av teamet for å høre hva de synes om "selvbestemt", så kan jeg lage en dedikert PR for å endre navn overalt. Så må også fikse den nye notifikasjondatabasen.